detox/src/DetoxExportWrapper.js

```diff
@@ -105,4 +104,14 @@ class DetoxExportWrapper {
   }
 }
 
+DetoxExportWrapper.prototype.globalCleanup = async function() {
+  try {
+    // TODO For the next consumer, need to come up with some kind of infra-code to allow for dynamic registration of cleanup-callbacks.
+    const GenyCloudDriver = require('./devices/drivers/android/genycloud/GenyCloudDriver');
+    await GenyCloudDriver.globalCleanup();
+  } catch (error) {
+    log.warn({ event: 'GLOBAL_CLEANUP' }, 'An error occurred trying to shut down Genymotion-cloud emulator instances!', error);
+  }
+}
```

Comment:
@noomorph if you have any better suggestions as to how to provide this in a cleaner way (leaving the TODO note aside), I'd be happy to hear them.

Reply:
Maybe we could add a link to the instance's web GUI, so it would be easy to open and examine it.
Also, it might be a good idea to add that link when a cloud emulator spins up.

Reply:
Yes, I will do both!